web/styles.css

```diff
@@ -115,6 +115,46 @@ h4 {
     margin-top: 20px;
 }
 
+.manifest-loader {
+    max-width: 600px;
+    margin: 20px auto;
+    text-align: left;
+}
+
+.input-group {
```

Comment:
Is there any way we could clean up our styles at some point by identifying any classes with identical attributes and consolidating them?